spring-ai-mcp-sample/src/main/resources/application.properties

```
@@ -0,0 +1,14 @@
+# Server port
+# server.port=8080
+
+# Logging
+# logging.level.org.springframework.ai.mcp=DEBUG
```

```suggestion
logging.level.org.springframework.ai.mcp=INFO
```
It make sense to show something in the logs

---

For the demo application when run into STDIO mode we should not print anything on the console in/out but the JSON RPC message part for the MCP protocol. 
Yet we can use file for the logging. 
Not sure yet what is the best way to disable the console logging in spring boot :)